shared/message/output/CompressedImage.proto

```diff
@@ -25,27 +25,85 @@ import "google/protobuf/timestamp.proto";
 import "Matrix.proto";
 import "Vector.proto";
 
+/**
+ * Contains lens and image output information.
```

Comment:
In general, I personally try to avoid using full stops in a comment like this unless you actually intend to make a new sentence. Many tools that use these sorts of comments to generate documentation work better when the text doesn't end in a full stop
```suggestion
 * Contains lens and image output information
```